docs/lsm/quickstart/quickstart.rst

```diff
@@ -253,7 +253,7 @@ The following command executes a script to copy the required resources to a spec
 
 .. code-block::
 
-   $ docker exec -ti -w /code clab-srlinux-inmanta-server  /code/setup.sh
+   $ docker exec -ti -w /code clab-srlinux-inmanta-server  /code/setup.sh <environment uuid>
```

Comment:
Does this code setup still work with the new disk layout?

Reply:
Yes, it should work with both layouts (cf https://github.com/inmanta/examples/commit/f66765bb1451c303bd7af811c9745b8f70f6a0cf#diff-96bee1b03fa3c4fbe209cd6258fea0b16687880a14d3c79247b3b30cba013b04)